From 6918e2cde9b03c084be49e450b04d96340707b30 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 25 Jul 2018 18:29:34 +0000
Subject: [PATCH] TRITON-606 ur-agent should be aware of alternate admin
 nictags (bump version) Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
 Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 371c404..2daffec 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-ur-agent",
   "description": "Ur",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "author": "Joyent (joyent.com)",
   "main": "./ur-agent",
   "private": true,
-- 
2.21.0

